profile/README.md

```diff
@@ -18,5 +18,7 @@ If you would like to be more up to date on the progress within Pulsar feel free
 * [Tentative Roadmap Discussion](https://github.com/orgs/pulsar-edit/discussions/2)
 * [Frequently Asked Questions](https://github.com/pulsar-edit/.github/blob/main/FAQ.md)
 * [Contributing Right Now](https://github.com/pulsar-edit/.github/blob/main/project-birth/CONTRIBUTING-DURING-START.md)
+* [Documentation Roadmap](https://github.com/pulsar-edit/pulsar-edit.github.io/issues/14)
+* [Hop Right In on Some PR's or Issues](https://github.com/pulls?q=is%3Aopen+user%3Apulsar-edit+archived%3Afalse+sort%3Acomments-asc)
 
 Otherwise you may always check out our [Discord Server](https://discord.gg/gp9UStzsHk).
```

Comment:
```suggestion
For other support feel free to join us here:
- [Discord Server](https://discord.gg/gp9UStzsHk).
- [Reddit](https://www.reddit.com/r/pulsaredit/)
```

Reply:
Oh this actually seems best to me, over PR #16 

How would @softcode589 feel about using this change rather than their PR?

Reply:
I don’t really know

Reply:
Fair enough, in that case apologies, but may go with this one, as its a bit more clear. @softcode589 

But this does still accomplish the task of properly linking to the subreddit